client/asset/eth/eth_test.go

```diff
@@ -937,7 +937,7 @@ func TestSwap(t *testing.T) {
 				t.Fatalf("%v: receipt coin value: %v != expected: %v",
 					testName, receipt.Coin().Value(), contract.Value)
 			}
-			if !bytes.Equal(receipt.Contract(), contract.SecretHash[:]) {
+			if !bytes.Equal(receipt.Contract()[4:], contract.SecretHash[:]) {
```

Comment:
Would you please quick update with the version check to ensure `Contract` is encoding it there as in https://github.com/decred/dcrdex/commit/bbc9d81b3697e328ca732c176913ff94f991b98a#diff-59ea012074143e9b443cb736910be9904ee1f5101bd081f609fd88b22e128491R942

```
			contractData := receipt.Contract()
			if swaps.AssetVersion != binary.BigEndian.Uint32(contractData[:4]) {
				t.Fatal("wrong contract version")
			}
			if !bytes.Equal(contractData[4:], contract.SecretHash[:]) 
```

before calling:

```
contractData := make([]byte, 36)
binary.BigEndian.PutUint32(contractData[:4], 12)
copy(contractData[4:], secretHash[:])
```